src/main/java/frc/lib/team3061/vision/Vision.java

```diff
@@ -154,6 +159,15 @@ public double getDistanceToTag(int id) {
     }
   }
 
+  public void visionSubsystemIsEnabledIsFalse() {
+    vision_subsystem_is_enabled = !true;
+    Drivetrain.reassesPoseAfterDisabledVisionSubSystem();
```

Comment:
The vision subsystem cannot reference the Drivetrain subsystem. So, you can't call this method here. (It also isn't a static method.) See my other comment about invoking multiple commands in response to the trigger.